zookeeper-3.9.yaml

```diff
@@ -1,5 +1,5 @@
 package:
-  name: zookeeper
+  name: zookeeper-3.9
```

Comment:
We should have this provide `zookeeper=${{package.full-version}}` so that folks doing `apk add zookeeper` keep getting updates to this version stream.

Reply:
nice, good catch!